harambee/views.py

```diff
@@ -720,6 +719,20 @@ def get_context_data(self, **kwargs):
         context["incorrect"] = incorrect_percentage
 
         level_passed = self.object.harambee_journey_module_rel.harambee.check_if_level_complete(self.object)
+        context["min_percent"] = self.object.harambee_journey_module_rel.harambee.get_percentage_required(self.object)
+        if level_passed:
+            context["message"] = "WELL DONE!"
+            context["level_passed"] = True
+        else:
+            context["message"] = "LEVEL COMPLETED"
```

Comment:
coveralls says this has not been called, ie there be no test for "LEVEL COMPLETED" which I find strange.